api/v1alpha1/release_types.go

```diff
@@ -54,9 +54,6 @@ const (
 	// ReleaseReasonPipelineFailed is the reason set when the release PipelineRun failed
 	ReleaseReasonPipelineFailed ReleaseReason = "ReleasePipelineFailed"
 
-	// ReleaseReasonReleasePlanValidationError is the reason set when there is a validation error with the ReleasePlan
```

Comment:
We already agreed long ago as part of HACBS-980 that there would be a separate error reported to the user for this. Why are you changing it as a PR to just move things around?